sp800-63d/vot_mapping.md

```diff
@@ -5,27 +5,29 @@
 ## Mapping Assurance Levels to Vectors of Trust
 _This section is normative._
 
-Vectors of Trust (VoT) is a standard for expressing information about an authentication transaction from an IdP to an RP. This information is split into several orthogonal component categories, much in the same way that the assurances defined in 800-63 are split into different xAL categories.
+Vectors of Trust (VoT) is a standard for expressing information about an authentication transaction from an IdP to an RP. This information is split into several orthogonal component categories, much in the same way that the assurance levels defined in 800-63 are split into different xAL categories.
 
 ### Trustmark
 
 The trustmark URI for these definitions is [[ URI for this document ]].
 
-### xAL to VoT Components
+### VoT Components
 
-This document uses three categories defined in VoT: Identity Proofing (P), Primary Credential Usage (C), and Assertion Presentation (A). These categories correspond to the Identity Assurance Level (IAL), Authenticator Assurance Level (AAL), and Federation Assurance Level (FAL), respectively. 
+This document uses the four components defined in VoT: Identity Proofing (P), Primary Credential Usage (C), Primary Credential Management (M), and Assertion Presentation (A).
 
-This document does not use the Primary Credential Management (M) category defined in VoT.
+The VoT components P, C, and A correspond to 800-63's Identity Assurance Level (IAL), Authenticator Assurance Level (AAL), and Federation Assurance Level (FAL), respectively. While M does not correspond directly to an 800-63 assurance level category, this document defines a set of M values derived from requirements and recommendations in 800-63. 
```

Comment:
As stated on call, this section needs to point out that we use the 800-63 terminology in here but reference the VoT terminology where appropriate. So for instance, we'll use "authenticator" here but it will still map to the "C" component, which stands for "Credential" in VoT, but we don't have to call it "Credential" here except for once.